FinniversKit/Sources/Fullscreen/FavoriteAdsList/FavoriteAdsListViewModel.swift

```diff
@@ -13,6 +13,8 @@ public struct FavoriteAdsListViewModel {
     public let emptyListViewTitle: String
     public let emptyListViewBody: String
     public let emptyListViewImage: UIImage
+    public var panelTextToriOnboarding: String
+    public var showPanelForToriOnboarding: Bool
```

Comment:
could it be better to just call it
* ` panelTextOnboarding`
* `showPanelForOnboarding`
?
This probably will be used for other brands in the future

Reply:
I'll change the var names 👍 Most likely we will have similar onboarding for other brands yes, but this one is very Tori-specific so most likely they will look different or live other places in the app.